Branch manual for Foreman 3.2 (#1949)
theforeman/theforeman.org: _includes/manuals/3.2/1.2_release_notes.md

## Release notes for {{page.version}}

### Headline features

#### Foreman on Debian 11 (Bullseye)

It is now possible to run Foreman on Debian 11. Users are encouraged to upgrade.

### Upgrade warnings

#### `require_ssl_smart_proxies` setting dropped

The `require_ssl_smart_proxies` setting has been dropped and Foreman now behaves as if the value was true (the default). This means it's no longer possible to use reverse DNS instead of client certificates over HTTPS connections. Upgrading users should ensure a valid Public Key Infrastructure (PKI) exists.

For more information see the [RFC](https://community.theforeman.org/t/drop-require-ssl-and-require-ssl-smart-proxies-settings/26772) and [PR](https://github.com/theforeman/foreman/pull/9021).

### Deprecations

#### Running Foreman on EL7

Foreman 2.1 introduced EL8 support and Katello 4.0 (on Foreman 2.4) followed.
While it's currently undecided when EL7 support will be dropped exactly, this is an early notice given the considerable number of EL7 deployments.

> For fresh installations, EL8 should considered the preferred target.

Reviewer: ```suggestion
For fresh installations, it is advisable to install on EL8.
```

Existing installations should start thinking about a migration plan.

> Note that this is only support to run Foreman and Foreman Proxy themselves on EL7. Managing EL7 systems remains supported. See [the RFC](https://community.theforeman.org/t/deprecation-plans-for-foreman-on-el7-debian-10-and-ubuntu-18-04/25008) for more information.

Reviewer: ```suggestion
Note that this support statement refers to running Foreman and Foreman Smart Proxy themselves on EL7. Managing EL7 client systems remains supported. See [the RFC](https://community.theforeman.org/t/deprecation-plans-for-foreman-on-el7-debian-10-and-ubuntu-18-04/25008) for more information.
```

#### Running Foreman on Debian 10

Now that Debian 11 is supported, Debian 10 support is deprecated and will be dropped with Foreman 3.4

> Note that this is only support to run Foreman and Foreman Proxy themselves on Debian 10. Managing Debia 10 systems remains supported. See [the RFC](https://community.theforeman.org/t/deprecation-plans-for-foreman-on-el7-debian-10-and-ubuntu-18-04/25008) for more information.

Reviewer: ```suggestion
Note that this support statement refers to running Foreman and Foreman Smart Proxy themselves on Debian 10. Managing Debian 10 client systems remains supported. See [the RFC](https://community.theforeman.org/t/deprecation-plans-for-foreman-on-el7-debian-10-and-ubuntu-18-04/25008) for more information.
```
